### PR TITLE
feat: redesign dimiru components for 2020 

### PR DESCRIPTION
**Pull request by uhmtoto**

### CODE REVIEW

**Comment by a1p4ca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dimigoin/dimigoin-front/pullRequest?prid=5065828&bid=16668672)

File: app/dimiru/components/DimiIcon.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    },
+    pointer: {
+      type: Boolean,
+      default: false
+    }
+  },
+  computed: {
+    computedStyle () {
+      return {
+        'cursor': this.pointer
```
Comment (a1p4ca) [inline app/dimiru/components/DimiIcon.vue]: Codacy Issue found: [Unnecessarily quoted property 'cursor' found.](https://app.codacy.com/manual/dimigoin/dimigoin-front/pullRequest?prid=5065828)

Comment (uhmtoto) [reply app/dimiru/components/DimiIcon.vue]: 고쳤어용

**Comment by a1p4ca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dimigoin/dimigoin-front/pullRequest?prid=5065828&bid=16668672)

File: app/dimiru/components/DimiIcon.vue (hunk 1 above)
Comment (a1p4ca) [inline app/dimiru/components/DimiIcon.vue]: Codacy Issue found: [Unnecessarily quoted property 'cursor' found.](https://app.codacy.com/manual/dimigoin/dimigoin-front/pullRequest?prid=5065828)

Comment (uhmtoto) [reply app/dimiru/components/DimiIcon.vue]: 헉 푸시가 안 돼 있었어요..

Comment (uhmtoto) [reply app/dimiru/components/DimiIcon.vue]: 진짜 고쳤어요 !

**Pull request merged**